<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajpojazdu</class>
 <widget class="QDialog" name="szukajpojazdu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QCheckBox" name="odRocznikaCB">
      <property name="text">
       <string>Od rocznika</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="numerRejCB">
      <property name="text">
       <string>Numer rejestracyjny</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="doRocznikaCB">
      <property name="text">
       <string>Do rocznika</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="markaLE"/>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="odRocznikSpinBox">
      <property name="minimum">
       <number>1900</number>
      </property>
      <property name="maximum">
       <number>2100</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="odDatyCB">
      <property name="text">
       <string>Od daty końca przeglądu</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="doDatyCB">
      <property name="text">
       <string>Do daty końca przeglądu</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="nrRejLE"/>
    </item>
    <item row="9" column="1">
     <widget class="QDateEdit" name="doDatyDE">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="markaCB">
      <property name="text">
       <string>Marka</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QDateEdit" name="odDatyDE">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="modelCB">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="doRocznikSpinBox">
      <property name="minimum">
       <number>1900</number>
      </property>
      <property name="maximum">
       <number>2100</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="modelLE"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
